### ISSUE TITLE
Conformance classes inconsistency

### ISSUE THREAD

**Issue by m-mohr** [stac-utils/stac-fastapi#371]
Some conformance classes have a trailing slash (sometimes before a fragment) and sometimes not.
For example sort is `https://api.stacspec.org/v1.0.0-beta.4/item-search/#sort` although it seems from the spec that it should be `https://api.stacspec.org/v1.0.0-beta.4/item-search#sort`

**Comment by m-mohr** [comment]
There seem to be a couple more things inconsistent: https://github.com/stac-utils/stac-fastapi/pull/417

**Comment by geospatial-jeff** [comment]
I think this may have been fixed but I'd like to do one more pass through all of the conformance classes after #432 is finished.

**Comment by gadomski** [comment]
I checked both backends locally and they look fine. I'm going to close this a fixed pre-2.4.4, please reopen if there's incorrect conformance classes found.

**Comment by m-mohr** [comment]
@gadomski Looks better now, but I just found that at least MSPC still uses strange CQL Text/JSON conformance classes.
Not sure whether that's a MS PC but or a stac fastapi bug though.

- MSPC: `https://api.stacspec.org/v1.0.0-rc.1/item-search#filter:cql-text` and json equivalent
- The Filter extension indicates: `http://www.opengis.net/spec/cql2/1.0/conf/cql2-text` and json equivalent

**Comment by gadomski** [comment]
Yup, I _think_ you're right but will ask Phil to check to make sure (over on the new issue).
